ui/lib/pki/addon/components/pki-generate-root.ts

```diff
@@ -71,6 +74,7 @@ export default class PkiGenerateRootComponent extends Component<Args> {
   get returnedFields() {
     return [
       'certificate',
+      ...parsedParameters,
```

Comment:
I considered adding these in other places - since we go through all this effort to parse the certificate values, it'd be nice to have them consistently show whenever there's a `certificate` param. 

But this would mean using the `PkiInfoTableRows` to consistently display data (b/c only common name is a value that will be potentially empty that should alwaysRender - all of the other attrs should only render if they have a value)